relay-server/src/service.rs

```diff
@@ -135,6 +135,9 @@ impl ServiceState {
         let outcome_producer = Arbiter::start(|_| outcome_producer);
         registry.set(outcome_producer.clone());
 
+        let outcome_aggregator = OutcomeAggregator::new(&config, outcome_producer.recipient());
+        registry.set(outcome_aggregator.start());
```

Comment:
Two questions:
1. Out of curiosity, why did you move this up here?
2. Why does the outcome aggregator not get its own thread?

Reply:
That probably wouldn't have been necessary, and I'm happy to revert that. I merely moved it up so that it's next to the outcome producer.

It doesn't have its own thread since it looks like it does relatively cheap stuff internally. I'm not yet convinced that we should have one thread for each actor/service, so I'm mostly going by what we know causes issues.